features/all/phpcr_query_update.feature

```diff
@@ -153,3 +153,30 @@ Feature: Execute a a raw UPDATE query in JCR_SQL2
             | UPDATE [nt:unstructured] mixin_foo('bar') |
             | UPDATE [nt:unstructured] APPLY mixin_foo('bar') |
             | UPDATE [nt:unstructured] mixin_foo'bar') |
+
+    Scenario Outline: Execute update query with expressions
+        Given I execute the "<query>" command
+        Then the command should not fail
+        And I save the session
```

Comment:
This scenario is really weird. ``I save the session`` is an actionable sentence. It should be in the WHEN part of the scenario, not the THEN part.